NearBeach/views/authentication_views.py

```diff
@@ -1,6 +1,6 @@
 # Import Forms
 from ..forms import PermissionSet, Group, LoginForm, User
-from ..models import UserGroup, Notification, Organisation, User
```

Comment:
@robotichead Do you want this to be `UserModel`?

Reply:
I think I made a mistake here. I've included the UserModel - and removed the "User" from the line above :)